lib/pages/portal/view/website_view.dart

```diff
@@ -336,16 +404,16 @@ class WebsiteIcon extends StatelessWidget {
       future: StorageProvider.findImageUrl('websites/${website.id}/icon.png'),
       //Firebase Storage path
       builder: (context, snapshot) {
-        ImageProvider image;
-        image = const AssetImage('assets/icons/globe.png');
+        ImageProvider imgOld;
```

Comment:
Here we make an additional quarry on storage that we don't need. Remove the Futter builder and let just circle image